To: vim_dev@googlegroups.com
Subject: Patch 7.3.313
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.313 (after 7.3.307)
Problem:    One more warning when compiling with dynamic Python 3.
Solution:   Change PySliceObject to PyObject.
Files:      src/if_python3.c


*** ../vim-7.3.312/src/if_python3.c	2011-09-14 15:41:54.000000000 +0200
--- src/if_python3.c	2011-09-14 18:54:49.000000000 +0200
***************
*** 194,200 ****
  static int (*py3_PySys_SetObject)(char *, PyObject *);
  static PyObject* (*py3_PyList_Append)(PyObject *, PyObject *);
  static Py_ssize_t (*py3_PyList_Size)(PyObject *);
! static int (*py3_PySlice_GetIndicesEx)(PySliceObject *r, Py_ssize_t length,
  		     Py_ssize_t *start, Py_ssize_t *stop, Py_ssize_t *step, Py_ssize_t *slicelength);
  static PyObject* (*py3_PyErr_NoMemory)(void);
  static void (*py3_Py_Finalize)(void);
--- 194,200 ----
  static int (*py3_PySys_SetObject)(char *, PyObject *);
  static PyObject* (*py3_PyList_Append)(PyObject *, PyObject *);
  static Py_ssize_t (*py3_PyList_Size)(PyObject *);
! static int (*py3_PySlice_GetIndicesEx)(PyObject *r, Py_ssize_t length,
  		     Py_ssize_t *start, Py_ssize_t *stop, Py_ssize_t *step, Py_ssize_t *slicelength);
  static PyObject* (*py3_PyErr_NoMemory)(void);
  static void (*py3_Py_Finalize)(void);
*** ../vim-7.3.312/src/version.c	2011-09-14 17:50:05.000000000 +0200
--- src/version.c	2011-09-14 18:56:03.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     313,
  /**/

-- 
All good vision statements are created by groups of people with bloated
bladders who would rather be doing anything else.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
